webextension/background/main.js

```diff
@@ -62,5 +63,19 @@ window.main = (function () {
     chrome.tabs.create({url: backend + "/shots"});
   });
 
+  communication.register("openShot", ({url, copied}) => {
```

Comment:
I'm not sure if main is the right place for this handler, but "openMyShots" is here so maybe its a decent spot.